_posts/2017-06-08-viewer-authentication-api.html

```diff
@@ -438,29 +239,27 @@ <h2>Hash lifetime and expiration</h2>
 
         <p>Expiration date has to be</p>
         <ul>
+            <li>Specified in UNIX timestamp format</li>
             <li>Part of the hash by concatenating it to the rest of the parameters</li>
             <li>Included in the response object as “hashExpire”.</li>
-            <li>Specified in UNIX timestamp format</li>
-            <li>Included in the response JSON object</li>
+            <li>JSON encoded as part of the response</li>
         </ul>
 
-        <p>An example array:</p>
+        <p>Example script in PHP for specifying hash expiration date</p>
 
-        <pre><code class="json">
-[
-    {
-        "userName": "testuser",
-    },
-    {
-        "someString": "someValue"
-    },
-    {
-        "hash": "56a8835175d33c368955dac6bdaf00de"
-    },
-    {
-        "hashExpire": 1333521590
-    }
-]
+        <pre><code class="php">
+// We specify an expiration date as a UNIX timestamp
+// which is valid for one hour.
+$expiration = time()+3600;
+
+// We concatenate it to the rest of the paramteres when creating the hash.
+$hash = md5(implode('|', $userData).'|'.$expiration.'|'.$SHARED_SECRET_KEY);
+
+// We inculde it in the response.
+$response = array_merge($userData, "hashExpire" => $expiration, "hash" => $hash);
```

Comment:
I don't think this code will work. The array syntax is not right in the array_merge